features/admin.feature

```diff
@@ -85,3 +85,30 @@ Feature: Admin
     And I am on the edit election page
     Then I should see 1 in "votedUsers"
     And I should see 1 in "activeUsers"
+
+  Scenario: Changing the card key of a user
+    Given There is an active user with card key "1234"
+    And I am waiting on page "/admin/change_card"
+    When I scan card key "1235"
+    And I fill in model "user.username" with "testuser"
```

Comment:
"I fill in model" describes functionality in the angular app not what the user does.